go/vt/sqlparser/normalizer_test.go

```diff
@@ -179,6 +179,20 @@ func TestNormalize(t *testing.T) {
 		in:      "select a, b from t order by 1 asc",
 		outstmt: "select a, b from t order by 1 asc",
 		outbv:   map[string]*querypb.BindVariable{},
+	}, {
+		// GROUP BY column_position
+		in:      "select a, b from t group by 1",
+		outstmt: "select a, b from t group by 1",
+		outbv:   map[string]*querypb.BindVariable{},
+	}, {
+		// ORDER BY column_position
```

Comment:
I think this comment is not correct?

Reply:
you are correct that the comment is incorrect